lib/redmine_pulls/patches/adapters/git_adapter_helper.rb

```diff
@@ -47,20 +44,48 @@ def mergable?(commit_base, commit_head)
             cmd_args << commit_base
             cmd_args << commit_head
 
-            merge_result = nil
-            git_cmd(cmd_args) { |io| io.binmode; merge_result = io.read }
+            merge_result = git_cmd_with_output(cmd_args)
+
+            # Split the regex in to two avoid conflict detection when working with this file
+            regex = Regexp.new("<<<" + "<<<<.*=======.*>>>>>>>", Regexp::MULTILINE)
+
+            ! regex.match(merge_result)
+          end
+
+          def merge(pull_number, commit_base, commit_head)
```

Comment:
- Change this method signature to `merge(commit_base, commit_head, options = {})`
- Move the commit message to `Pull#merge_commit_message`
- The description should then be `options.message || "Merge #{commit_head} into #{commit_base}"`
- The author's name and email should be passed to merge in a similar way.